etc/insights-client.motd

```diff
@@ -0,0 +1,2 @@
+Missing out on Insights benefits. To register run: insights-client --register
```

Comment:
Needs to communicate the system status more clearly, and also the benefits of Insights.
https://www.redhat.com/en/technologies/management/insights is probably the place that has that written up best.

What about something like:
```
This system is not registered to Red Hat Insights.
https://www.redhat.com/en/technologies/management/insights

To register this system, run: insights-client --register
```

Reply:
Possibly https://cloud.redhat.com is a better (and for sure much more stable URL).
This is what we went for in https://github.com/candlepin/subscription-manager/pull/2089

Reply:
Thanks. Updated the message.